code/processes/gateway.q

```diff
@@ -450,7 +450,8 @@ syncexecjt:{[query;servertype;joinfunction;timeout]
  // doesn't make sense to allow specification of a callback for sync requests
  asyncexecjpts[query;servertype;joinfunction;();timeout;1b];
  // defer response
- @[-30!;(::);()];
+ e:@[{-30!x;1b};(::);0b];
+ if[not e;[.lg.o[`syncexecjt;"failed to defer; query passed to syncexecjpre36"];:syncexecjpre36[query;servertype;joinfunction]]] 
```

Comment:
```suggestion
 if[not e;
     .lg.o[`syncexecjt;"failed to defer; query passed to syncexecjpre36"];
     :syncexecjpre36[query;servertype;joinfunction];
  ];
```

Don't need to use a block (square brackets) inside an `if`, and a bit more readable if split over multiple lines